.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @dylanj1752 @felddy @INLGuy @itsmostafa @jsf9k @mcdonnnj @xlyk
+* @dav3r @dylanj1752 @felddy @INLGuy @itsmostafa @izzy64 @jsf9k @mcdonnnj @xlyk
```

Comment:
yup ✅

Reply:
@itsmostafa I'm going to wait for the resolution in https://github.com/cisagov/con-pca-api/pull/811#discussion_r1000789552 since that applies here as well.

Reply:
@mcdonnnj sounds good